epoxy-sample/src/main/java/com/airbnb/epoxy/SampleAdapter.java

```diff
@@ -32,21 +40,30 @@
         .title(R.string.epoxy)
         .caption(R.string.header_subtitle);
 
+    EditTextModel editTextModel = new EditTextModel_()
+        .editTextListener(editTextListener)
+        .hint(R.string.text_hint);
+
     ButtonModel addButton = new ButtonModel_()
         .text(R.string.button_add)
+        .backgroundColor(ContextCompat.getColor(context, R.color.green))
```

Comment:
You don't need to pass a context to the models, `bind` passes a view so just do `view.getContext()`